<?xml version="1.0"?>
<launch>
    <arg name="controllers" default="bspline_ff_kino_joint_trajectory_controller"/>

    <rosparam file="$(find kinodynamic_neural_planner)/config/config.yaml" command="load" subst_value="true"/>

    <node pkg="kinodynamic_neural_planner" type="neural_bspline_planner_node.py" name="neural_bspline_planner_node" output="screen">
        <param name="controllers" value="$(arg controllers)" />
    </node>

<!--     <node pkg="kinodynamic_neural_planner" type="baseline_planner_node.py" name="baseline_planner_node" output="screen"> -->
<!--         <param name="controllers" value="$(arg controllers)" /> -->
<!--     </node> -->

</launch>
